config/locales/en.yml

```diff
@@ -30,4 +30,11 @@
 # available at https://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  need_signup_for_access: "You need to be signed in to access that."
+  account_creation_successful: "Account created successfully"
+  successful_sign_in: "Sign in successful"
+  wrong_email_password: "Wrong email or password"
+  signed_out: "Signed out"
+  user_already_signed_in: "User has already signed in"
+  need_admin_for_access: "You need to be Admin to access that."
+  not_found: ' not found'
```

Comment:
you can break the translations into sections like: error, success, sign_up, sign_in and so on